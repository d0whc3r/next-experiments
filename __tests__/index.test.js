/* global it, expect, describe */

import React from 'react';
import renderer from 'react-test-renderer';
import App from '../pages/index.js';

describe('With Snapshot Testing', () => {
  it('App shows "Hello world!"', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
});
