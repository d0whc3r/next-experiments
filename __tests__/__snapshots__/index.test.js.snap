// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
<wrapper>
  <nav
    id="navigationbar"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
     |
    <a
      href="/"
      onClick={[Function]}
    >
      index2
    </a>
     |
    <a
      href="/about"
      onClick={[Function]}
    >
      About
    </a>
     |
    <a
      href="/about/a2"
      onClick={[Function]}
    >
      About 2
    </a>
    <a
      href="./a2"
      onClick={[Function]}
    >
      About
    </a>
  </nav>
  <div
    id="root"
  >
    <div>
      Hello World. 
      <a
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
    </div>
    <button
      className="ui primary button"
      onClick={[Function]}
      tabIndex={undefined}
    >
      Button
    </button>
    <button
      className="ui secondary button"
      onClick={[Function]}
      tabIndex={undefined}
    >
      Button
    </button>
    <button
      className="ui button"
      onClick={[Function]}
      tabIndex={undefined}
    >
      Button
    </button>
    <button
      className="ui green small button"
      onClick={[Function]}
      tabIndex={undefined}
    >
      <i
        aria-hidden="true"
        className="download icon"
      />
      Download
    </button>
    <li>
      <a
        href="/a"
        onClick={[Function]}
      >
        a
      </a>
    </li>
    <li>
      <a
        href="/b"
        onClick={[Function]}
      >
        b
      </a>
    </li>
  </div>
  <div
    id="footer"
  >
    footer
  </div>
</wrapper>
`;
